.github/workflows/perform-release.yml

```diff
@@ -0,0 +1,142 @@
+name: "Perform Release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_pr_number:
+        description: "The PR number of the release PR"
+        required: true
+      skip-pr-merge:
+        description: "Whether to skip merging the PRs"
+        required: false
+        default: false
+        type: boolean
+
+env:
+  MVN_CLI_ARGS: --batch-mode --no-transfer-progress --fail-at-end --show-version -DskipTests
+  JAVA_VERSION: 17
+
+jobs:
+  prerequisites:
+    name: "Prerequisites"
+    outputs:
+      release-version: ${{ steps.determine-branch-names.outputs.RELEASE_VERSION }}
+      release-tag: ${{ steps.determine-branch-names.outputs.RELEASE_TAG }}
+      code-branch: ${{ steps.determine-branch-names.outputs.CODE_BRANCH_NAME }}
+      release-commit: ${{ steps.determine-branch-names.outputs.RELEASE_COMMIT }}
+    permissions:
+      pull-requests: read
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Determine Branch Names"
+        id: determine-branch-names
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request_review" ]]; then
+            echo "[DEBUG] Taking branch name from pull request event"
+            BRANCH_NAME=${{ github.event.pull_request.head.ref }}
+          elif [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            echo "[DEBUG] Taking branch name from workflow dispatch event"
+            BRANCH_NAME=$(gh pr view ${{ github.event.inputs.release_pr_number }} --repo "${{ github.repository }}" --json headRefName | jq -r '.headRefName')
+          else
+            echo "Cannot determine branch name from event '${{ github.event_name }}'"
+            exit 1
+          fi
+          
+          RELEASE_VERSION=$(echo "$BRANCH_NAME" | cut -d '-' -f2) 
+
+          echo "RELEASE_VERSION=$RELEASE_VERSION" >> $GITHUB_OUTPUT
+          echo "RELEASE_TAG=rel/$RELEASE_VERSION" >> $GITHUB_OUTPUT
+          echo "CODE_BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_OUTPUT
+          echo "RELEASE_COMMIT=$(gh release view "$RELEASE_TAG" --repo "${{ github.repository }}" --json targetCommitish | jq -r '.targetCommitish')" >> $GITHUB_OUTPUT
+          
+          echo "[DEBUG] Current GITHUB_OUTPUT: '$(cat $GITHUB_OUTPUT)'"
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - name: "Checkout Repository"
+        uses: actions/checkout@v4
+
+      - name: "Check Whether Code PR Can Be Merged"
+        if: ${{ inputs.skip-pr-merge != 'true' }}
+        uses: ./.github/actions/pr-is-mergeable
+        with:
+          pr-ref: ${{ steps.determine-branch-names.outputs.CODE_BRANCH_NAME }}
+          excluded-check-runs: |
+            {
+              \"Continuous Integration\": [\"Run BlackDuck Scan\", \"Run Security Rating\"], 
```

Comment:
(Minor/Question)

This is a leftover from copying right? We don't use Blackduck in this repo, AFAIK.